vcd_cli/disk.py

```diff
@@ -209,3 +211,30 @@ def update(ctx, name, size, description, new_name, storage_profile, disk_id):
         stdout(task, ctx)
     except Exception as e:
         stderr(e, ctx)
+
+@disk.command('change-owner', short_help='change owner of disk')
+@click.pass_context
+@click.argument('disk-name',
+                metavar='<disk-name>',
+                required=True)
+@click.argument('user_name',
+                metavar='<user_name>',
+                required=True)
+@click.option('-i',
+              '--id',
+              'disk_id',
+              required=False,
+              metavar='<id>',
+              help='Disk id')
+def change_disk_owner(ctx, disk_name, user_name, disk_id):
```

Comment:
I have a slight issue with how we are ordering params in the function definitions.
Java param ordering convention : Params that help identify the resource (which are also generally final) preceed the params that hold the values that needs to be updated.

Reply:
I have put the required argument first and then optional. Lets discuss and finalize on a standard that everyone has to follow.